### ISSUE TITLE
Make it compatible with PEP-567

### ISSUE THREAD

**Issue by fantix** [fantix/aiocontextvars#2]
[PEP-567](https://www.python.org/dev/peps/pep-0567/) is accepted for Python 3.7. We should make this library compatible.

**Comment by fantix** [comment]
In c33d9fc31a440653bdc7fbc53552d24813a5f6b7 I replaced the local implementation with upstream `contextvars` library, and added partial asyncio support.

**Comment by jjjacksn** [comment]
@fantix will you be cutting a 0.2.0 release soon?

**Comment by fantix** [comment]
Yes! Will do as soon as test is fixed.

**Comment by jjjacksn** [comment]
Do you mean the [failing travis build for master](https://travis-ci.org/fantix/aiocontextvars/builds/424615676) or some specific test failures? I believe there is an issue with python 3.7 in Travis CI's default configuration -- more specifically with Python 3.7 and Ubuntu 14.04. https://github.com/travis-ci/travis-ci/issues/9815 describes a workaround. If you update .travis.yml to contain something like this, the build may pass.

```yaml
python: 
  - 3.5
  - 3.6

matrix:
  include:
    - python: 3.7
      dist: xenial
      sudo: true
```

I'll go ahead and open a PR with this change.

**Comment by fantix** [comment]
Yes that’s the one you fixed. Thanks a lot! I’ll push a release as soon as I’m back to my keyboard.
